update `bin/configure` to prohibit empty application name

Natblow opened the pull request:
On my first time using of Bullet Train, I've opened an issue #607 where I could resolve later the same day. 
I must have put an empty string for some reason when prompted about the name of an application which caused that issue. 

This PR is to prevent this problem if it occurs.

Also I have added `[y/n]` on the first question prompt.

kaspth commented:
Sweet! I glanced over the file and it looks like `ssh_path` can have the same issue: https://github.com/bullet-train-co/bullet_train/blob/d0ea533c4bc3bc25739f69610f1b5f656587587d/bin/configure#L145

I think we should fix that one too.

It looks like several of the `ask` lines could use the `[y/n]` treatment, can we split that into a separate PR and update all of those?

Natblow commented:
Thanks @kaspth , those are great points! 
I now added the same restriction for the `ssh_path` value.

I've also added `[y/n]` options on all ask lines where necessary, on a separate PR #613

kaspth commented:
@Natblow sweeeet, thank you so much!
